src/EA8AE0.c

```diff
@@ -24,7 +24,30 @@ static char* N(exit_str_2) = "kzn_20";
 
 INCLUDE_ASM(s32, "EA8AE0", func_80240B00_EA95E0);
 
-INCLUDE_ASM(s32, "EA8AE0", func_80240BD4_EA96B4);
+typedef struct Unk80240BD4 {
+    f32 unk_0;
+    f32 unk_4;
+    f32 unk_8;
+} Unk80240BD4;
+
+ApiStatus func_80240BD4_EA96B4(Evt *script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 temp_s0_5;
```

Comment:
this var can be removed, as can `temp` below

Reply:
done